sdk.yaml

```diff
@@ -396,6 +396,16 @@ REST:
       Member List:
         .documentation: https://ably.com/docs/rest/presence#member-list
         .specification: RSP3
+    Publish:
+      .specification: RSL1
+      .synopsis: |
+        Send a message on this channel.
```

Comment:
Something to probably discuss with docs / marketing, but should we always describe this as "publish a message to a channel"?

Reply:
I know, in part, that I would have wanted to use a word that was different to 'publish' - given that's the name of the feature node. However, if there's potential for misinterpretation with 'send' vs 'publish' then I will happily change it.

Reply:
> if there's potential for misinterpretation

I don't think there's potential for misinterpretation, I'd just like us to use consistent terminology, so it's always "Publish a message to" rather than "Send a message on", but I don't know if that aligns with docs / marketing, so if you don't think it matters, then nor do I 😉